api/app/signals/apps/reporting/csv/horeca.py

```diff
@@ -239,5 +255,11 @@ def create_csv_files(isoweek, isoyear, save_in_dir=None):
             for csv_file in csv_files:
                 logger.info('Copy file "{}" to "{}"'.format(csv_file, save_in_dir))
                 shutil.copy(csv_file, save_in_dir)
+        elif os.getenv('SWIFT_ENABLED', 'false') == 'true':
```

Comment:
Why a string? This should be a boolean, right?

Reply:
Env variables have no type Boolean, hence the use of strings.